tests/unit/teardown-context-test.js

```diff
@@ -69,4 +76,16 @@ module('teardownContext', function(hooks) {
       assert.equal(state.pendingRequestCount, 0, 'pendingRequestCount is 0');
     });
   }
+
+  test('can opt out of waiting for settledness', async function(assert) {
+    later(() => assert.step('later'), 200);
```

Comment:
looks like a potentially flaky test to me due to the fixed timeout. Maybe RSVP.defer() + custom waiter would be better?

Reply:
Ya, I'll swap to a waiter (RSVP.defer also has issues).